cmd/release-controller/sync_verify_prow.go

```diff
@@ -22,7 +22,23 @@ import (
 
 func (c *Controller) ensureProwJobForReleaseTag(release *Release, verifyName string, verifyType ReleaseVerification, releaseTag *imagev1.TagReference, previousTag, previousReleasePullSpec string, extraLabels map[string]string) (*unstructured.Unstructured, error) {
 	jobName := verifyType.ProwJob.Name
+
+	isAggregatedJob := false
+	if verifyType.AggregatedProwJob != nil && verifyType.AggregatedProwJob.AnalysisJobCount > 0 {
+		isAggregatedJob = true
+
+		// Provide a way to override the default aggregator image
+		if verifyType.AggregatedProwJob.ProwJob != nil && len(verifyType.AggregatedProwJob.ProwJob.Name) > 0 {
+			jobName = verifyType.AggregatedProwJob.ProwJob.Name
+		} else {
+			jobName = defaultAggregateProwJobName
```

Comment:
as a name, this looks not awesome.  What is the "gcp-sdn-zstream-upgrade" called on this line  https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.10-ci.json#L66 ?

Naming the aggregator for that seems to make more sense.  If its easier, let's make that prowjob.name required.

Reply:
oh, or prowjob.name of the parent + "aggregated".  that's probably more consistent, right?

Reply:
So, `gcp-sdn-zstream-upgrade` is the `verifyName`.  For instance, when the release `4.9.0-0.nightly-2021-09-09-184742` gets created, you'd see this subset of prowjobs for the analysis and aggregator:
```
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-aws
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-gcp-sdn-zstream-upgrade-aggregator
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-gcp-sdn-zstream-upgrade-analysis-1
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-gcp-sdn-zstream-upgrade-analysis-2
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-gcp-sdn-zstream-upgrade-analysis-3
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-gcp-sdn-zstream-upgrade-analysis-4
prowjob.prow.k8s.io/4.9.0-0.nightly-2021-09-09-184742-gcp-sdn-zstream-upgrade-analysis-5
```